test/integration/manifests/cnsconfig/overlayconfigmap.yaml

```diff
@@ -29,3 +29,37 @@ data:
       "CNIConflistFilepath": "/etc/cni/net.d/05-cilium.conflist",
       "CNIConflistScenario": "cilium"
     }
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cns-win-config
+  namespace: kube-system
+data:
+  cns_config.json: |
+    {
+      "TelemetrySettings": {
+          "TelemetryBatchSizeBytes": 16384,
+          "TelemetryBatchIntervalInSecs": 15,
+          "RefreshIntervalInSecs": 15,
+          "DisableAll": false,
+          "HeartBeatIntervalInMins": 30,
+          "DebugMode": false,
+          "SnapshotIntervalInMins": 60
+      },
+      "ManagedSettings": {
+          "PrivateEndpoint": "",
+          "InfrastructureNetworkID": "",
+          "NodeID": "",
+          "NodeSyncIntervalInSeconds": 30
+      },
+      "EnableSubnetScarcity": false,
+      "ChannelMode": "CRD",
+      "InitializeFromCNI": true,
+      "ProgramSNATIPTables": false,
+      "ManageEndpointState": false,
+      "MetricsBindAddress": ":10092",
+      "EnableCNIConflistGeneration": false,
+      "CNIConflistFilepath": "C:\\k\\azurecni\\etconf\\10-azure.conflist",
```

Comment:
should it be C:\\k\\azurecni\\**netconf**\\10-azure.conflist

Reply:
Yes I think you're right

Reply:
One thing, is I think dropgz should be installing the conflist, at least like it does for linux today:
```
initContainers:
  - args:
    - deploy
    - azure-vnet
    - -o
    - /opt/cni/bin/azure-vnet
    - azure-vnet-telemetry
    - -o
    - /opt/cni/bin/azure-vnet-telemetry
    - azure-swift-overlay.conflist
    - -o
    - /etc/cni/net.d/15-azure-swift-overlay.conflist
    command:
    - /dropgz
```

Reply:
yeah, dropgz should be able to install conflist on windows as well.